src/main/java/org/opensrp/service/EventService.java

```diff
@@ -402,8 +402,9 @@ public Event mergeEvent(Event updatedEvent) {
 		return allEvents.findEvents(eventSearchBean);
 	}
 
-	@PreAuthorize("hasRole('EVENT_VIEW')")
-	@PostFilter("hasPermission(filterObject, 'EVENT_VIEW')")
+	/*
```

Comment:
can we add more details on why this is being excluded, also the exclusion should be via a new method